src/scripts/main.js

```diff
@@ -1,3 +1,248 @@
 'use strict';
 
-// write code here
+/* eslint-disable */
+const body = document.querySelector('body');
+const tHeadCells = document.querySelectorAll('table thead th');
+const tbody = document.querySelector('table tbody');
+const rows = [...tbody.querySelectorAll('tr')];
+
+const officeOptions
+  = ['Tokyo', 'Singapore', 'London', 'New York', 'Edinburgh', 'San Francisco'];
+
+employeeForm();
+
+tHeadCells.forEach((cell, index) => {
+  cell.addEventListener('click', () => {
+    const receivedRows = sortByIdAndType(index);
+    receivedRows.forEach(row => tbody.append(row));
+  })
+})
+
+function sortByIdAndType(index) {
+  const currentRows = [...tbody.querySelectorAll('tr')];
+  const rows = [...currentRows];
+
+  rows.sort((rowA, rowB) => {
+    const cellA = rowA.cells[index].innerText;
+    const cellB = rowB.cells[index].innerText;
+
+    return cellA.localeCompare(cellB, 'en', { numeric: true });
+  });
+
+  if (JSON.stringify(rows.map(row => row.innerText))
+    === JSON.stringify(currentRows.map(row => row.innerText))) {
```

Comment:
Id recommend to move such big conditions to own variables for better readability. Check similar cases